src/network/http_server.rs

```diff
@@ -1702,16 +1890,7 @@ impl HubHttpService for HubHttpServiceImpl {
     async fn get_casts_by_fid(&self, req: FidRequest) -> Result<PagedResponse, ErrorResponse> {
         let service = &self.service;
         let response = service
-            .get_all_cast_messages_by_fid(tonic::Request::<FidTimestampRequest>::new(
-                FidTimestampRequest {
-                    fid: req.fid,
-                    page_size: req.page_size,
-                    page_token: req.page_token,
-                    reverse: req.reverse,
-                    start_timestamp: req.start_timestamp,
-                    stop_timestamp: req.stop_timestamp,
-                },
-            ))
+            .get_casts_by_fid(tonic::Request::<proto::FidRequest>::new(req.to_proto()))
```

Comment:
This is a hubble bug. All other endpoints call the respective get_<type>_by_fid method, not get_all_<type> which also returns removes. Fixed to be consistent.

Reply:
Hmm, actually, people are relying on this behavior. Will revert to keep existing functionality.